copyright

```diff
@@ -220,6 +221,18 @@ License: public-domain
  uploaded to unsplash.com before June 2017, so it is
  in the public domain.
 
+Files:
+ images/land/station49*
+Copyright: Andrea De Santis
+License: Unsplash
+ Taken from unsplash.com and cropped.
+
+Files:
+ images/land/station50*
+Copyright: Mariia Ilina
+License: Unsplash
+ Taken from unsplash.com, rotated and cropped.
+
```

Comment:
This should be moved to below station48*, as right now station4* is below this and makes the copyright file invalid. These should also both have descriptions of why they have the Unsplash license and are not public domain, along with a link to an archived version of the page so the publish date can be verified.
```suggestion
Files:
 images/land/station49*
Copyright: Andrea De Santis
License: Unsplash
Comment: Taken from unsplash.com and cropped.

Files:
 images/land/station50*
Copyright: Mariia Ilina
License: Unsplash
Comment: Taken from unsplash.com, rotated and cropped.

```